scripts/windows/test/Run-E2ETest.ps1

```diff
@@ -786,11 +786,15 @@ Function RunTransparentGatewayTest
     TestSetup
 
     $testStartAt = Get-Date
-    $deviceId = "e2e-${ReleaseLabel}-Windows-${Architecture}-TransGW"
-    PrintHighlightedMessage "Run quickstart test with -d ""$deviceId"" started at $testStartAt."
+    $edgeDeviceId = "e2e-${ReleaseLabel}-Windows-${Architecture}-TransGW"
+    PrintHighlightedMessage "Run quickstart test with -d ""$edgeDeviceId"" started at $testStartAt."
 
-    # setup certificate chain to create the Edge device and leaf device certificates
-    PrepareCertificateChain
+    # setup certificate generation tools to create the Edge device and leaf device certificates
+    PrepareCertificateTools
+    # dot source the certificate generation script
```

Comment:
nit: suggest to move this line inside PrepareCertificateTools method.

Reply:
I agree with you, but the problem with that found that I could not invoke subsequent commands in the script unless it was dot sourced in the function. if you have a suggestion on how to fix, I am all ears.